freeipa-api/src/main/java/com/sequenceiq/freeipa/api/v1/freeipa/stack/model/common/Status.java

```diff
@@ -117,4 +138,37 @@ public boolean isVerticallyScalable() {
         return !FREEIPA_VERTICALLY_NON_SCALABLE_STATUSES.contains(this);
     }
 
+    //CHECKSTYLE:OFF: CyclomaticComplexity
```

Comment:
Sanity check: Is this needed? If so, there is no corresponding comment to re-enable the rule validation.